tedana/io.py

```diff
@@ -404,3 +406,91 @@ def load_data(data, n_echos=None):
     ref_img.header.set_sform(ref_img.header.get_sform(), code=1)
 
     return fdata, ref_img
+
+
+def add_decomp_prefix(name, prefix, max_value):
```

Comment:
Is this one supposed to be private, too ?

Reply:
If not, can we fill out the docstring ?

Reply:
It should be public. thanks for catching this.